sci-mathematics/scilab/files/scilab-5.5.2-batik-1.8.patch

```diff
@@ -0,0 +1,134 @@
+diff --git a/scilab/configure b/scilab/configure
+index fd0e339..a4451ce 100755
+--- a/scilab/configure
++++ b/scilab/configure
```

Comment:
Done.  Thanks for the fast review and for getting the ball rolling previously, Marius.